job_templates/service_action_-_enable_web_console.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+name: Service Action - Enable Web Console
+job_category: Ansible Services
+snippet: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+---
+- hosts: all
+  tasks:
+  - name: ensure cockpit is installed
+    package:
+      name: "cockpit"
+      state: present
+
+  - name: enable firewall rule for cockpit
```

Comment:
Would it make sense to add a parameter to do this? Not everyone is using firewalld.